conf/messages.en

```diff
@@ -465,26 +465,58 @@ pdf.ruling.aboutDecision.text=I have made this decision under section 24 of The
+pdf.ruling.section.askForReview.solsTeam.4=any new information,
+pdf.ruling.section.askForReview.solsTeam.5=reasons why you disagree with the decision.
+pdf.ruling.section.askForReview.solsTeam.cannotWrite=If there is a reason why you cannot write to the SOLS team within 30 days from the date of this letter, you will need a reasonable excuse for the delay to obtain an extension. You will need to write to the SOLS team as soon as possible to explain the delay in accepting the offer of a review. If you do not have a reasonable excuse, the review cannot be undertaken. You can ask a tribunal to accept a late appeal.
+pdf.ruling.section.askForReview.solsTeam.email=reviews@hmrc.gov.uk
+pdf.ruling.section.askForReview.solsTeam.emailText=The SOLS team email address is: <a class="govuk-link" href="mailto:{0}">{1}</a> If you decide to use email to correspond, please see the important information at the end of this letter. If you want to contact the team in writing the address is –
+pdf.application.footer.solicitors.office.and.legal.services.address=Solicitors Office and Legal Services<br> HM Revenue and Customs<br> BX9 1ZT
+
+pdf.ruling.section.appealForIndependentTribunal.title=If you want to appeal to an independent tribunal
+pdf.ruling.section.appealForIndependentTribunal.appealImmediately=If you would prefer, you can appeal immediately to an independent tribunal. You can also progress to a tribunal appeal once you have received the review outcome and disagree with the decision.
+pdf.ruling.section.appealForIndependentTribunal.courtsAndTribunals=Your request must reach HM Courts and Tribunals service within 30 days of the date of this letter, or within 30 days of the date of the letter that tells you the outcome of the review.
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements=The request should include copies of:
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements.1=this letter, or review outcome letter
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements.2=any assessment
+pdf.ruling.section.appealForIndependentTribunal.relevantInfo=If you do not send the relevant information, HM Courts and Tribunals may reject your appeal.
+pdf.ruling.section.appealForIndependentTribunal.tribunalsInfo=For more information about tribunals go to the page below or call 0300 123 1024.
+pdf.ruling.section.appealForIndependentTribunal.page=<a href="www.gov.uk/tax-tribunal" target="_blank">www.gov.uk/tax-tribunal</a>
+
+pdf.ruling.section.appealsAndReviews.title=More information about appeals and reviews
+pdf.ruling.section.appealsAndReviews.moreInfo=For more information about your appeal and review rights, go to the following page:
+pdf.ruling.section.appealsAndReviews.page=<a href="www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision" target="_blank">www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision</a>
```

Comment:
Same here, the href is wrong and redirecting to a not found page. Please use the full url and update the href to `https://www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision`. Also please add `class="govuk-link"` within the anchor tag for this link, thanks